src/components/_global/BalVerticalSteps/BalVerticalSteps.vue

```diff
@@ -0,0 +1,108 @@
+<script lang="ts" setup>
+import { useClassName } from '@/components/utils';
```

Comment:
I think we should reserve the `use` prefix for composables. 

To me, this function is like a switch/toggle/active... maybe we call it `switchClassName` or `toggleClassName` or `getActiveClassName`, etc.